tests/ui/lint/lint-unexported-no-mangle.stderr

```diff
@@ -31,7 +31,7 @@ error: const items should never be `#[no_mangle]`
   --> $DIR/lint-unexported-no-mangle.rs:9:1
    |
 LL | const FOO: u64 = 1;
-   | -----^^^^^^^^^^^^^^
+   | ------^^^^^^^^^^^^^
```

Comment:
I think this breaks the suggestion. I think applying the suggestion will now result in `pub staticFOO: u64 = 1;` with the space missing.